gradle.properties

```diff
@@ -19,6 +19,8 @@ httpClientVersion=4.5.1
 jedisVersion=2.8.1
 h2Version=1.4.191
 springDataMongoVersion=1.9.1.RELEASE
+springDataCassandraVersion=1.5.0.M1
```

Comment:
Lowered to the stable version, which also drops the Cassandra version from 3 to 2.

Tests fail on cass3, but pass on cass2 locally. 

Pretty sure this means the travis tests will be broken until I can get embedded cass working.